src/config/services/servicesConfig.go

```diff
@@ -36,6 +38,7 @@ type Config struct {
 	Twitter         Twitter
 	RSS             RSS
 	AssetLinks      AssetLinks
+	Aasa            Aasa
```

Comment:
This needs to be all caps because it's an acronym. See `RSS` on line 39. If this was starting with a lowercase letter, then depending on the content this would be fine. Because this is an exported field, it starts with a capital which means the entire thing needs to be.